.gitignore

```diff
@@ -86,10 +86,16 @@ ENV/
 vrenv/my_env
 tests/get-pip.py
 tests/samples.py
-
```

Comment:
Your .gitignore should only have the relevant files to your project ignored. It's considered part of the documentation. Consider removing the Spyder project settings section. and Rope project settings